internal/app/mymemory/mymemory.go

```diff
@@ -0,0 +1,69 @@
+package mymemory
+
+import (
+	"crypto/rand"
+	"fmt"
+)
+
+type MyMemory struct {
+	myMemory map[string]string
+}
+
+func InitMyMemory(initValue map[string]string) *MyMemory {
+	m := MyMemory{}
+	m.myMemory = initValue
+
+	return &m
+}
+
+func (m *MyMemory) CreateOrGetFromMemory(url []byte) (string, error) {
```

Comment:
It would be better if you use `url` as `string`

Reply:
corrected, nice comment!